deploy/bundle/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -2407,7 +2407,7 @@ spec:
                   value: noobaa/noobaa-core:5.7.0-20201216
                 - name: NOOBAA_DB_IMAGE
                   value: centos/postgresql-12-centos7
-                image: quay.io/ocs-dev/ocs-operator:latest
+                image: quay.io/pkundra/ocs-operator:latest
```

Comment:
same comment. don't merge with this. I guess this is for your testing?

Reply:
yes.